src/main/java/Duke.java

```diff
@@ -1,10 +1,172 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
 public class Duke {
+
+    public static final int LENGTH_DEADLINE = 9;
+    public static final int LENGTH_EVENT = 6;
+    public static final int LENGTH_TODO = 5;
+    public static final int SIZE_DONE_COMMAND = 2;
+
+    public static void getDateTime() {
+        LocalDateTime myDateObj = LocalDateTime.now();
+        DateTimeFormatter myFormatObj = DateTimeFormatter.ofPattern("dd-MM-yyyy HH:mm:ss");
+        String formattedDate = myDateObj.format(myFormatObj);
+        System.out.println(formattedDate);
+    }
+
+    public static void completeTask(ArrayList<Task> tasks, int taskIndex) {
+        taskIndex--; // index starts from 0, unlike listing number
+        if ( (taskIndex < tasks.size()) || (taskIndex > 0)) { // check if out of bounce
```

Comment:
There is a coding standard violation that between the first two barcekets, there should be no space.